internal/languages/python/analyzer/analyzer.go

```diff
@@ -35,10 +35,12 @@ func (analyzer *analyzer) Analyze(node *sitter.Node, visitChildren func() error)
 		return analyzer.analyzeSubscript(node, visitChildren)
 	case "call":
 		return analyzer.analyzeCall(node, visitChildren)
-	case "argument_list":
-		return analyzer.analyzeGenericOperation(node, visitChildren)
-	case "expression_statement":
+	case "argument_list", "expression_statement", "list", "tuple":
 		return analyzer.analyzeGenericOperation(node, visitChildren)
+	case "parenthesized_expression":
+		return analyzer.analyzeGenericConstruct(node, visitChildren)
+	case "keyword_argument":
+		return analyzer.analyzeKeywordArgumentOperation(node, visitChildren)
```

Comment:
WDYT about dropping "Operation" from the name?

```suggestion
		return analyzer.analyzeKeywordArgument(node, visitChildren)
```